ext/sockets/tests/socket_cmsg_credentials_fbsd.phpt

```diff
@@ -0,0 +1,92 @@
+--TEST--
+recvmsg(): receive SCM_CREDS messages
+--EXTENSIONS--
+sockets
+--SKIPIF--
+<?php
+
+if (!str_contains(PHP_OS, 'FreeBSD')) {
+    die('supported only on FreeBSD');
+}
+--CLEAN--
+<?php
+$path = __DIR__ . "/socket_cmsg_credentials_fbsd.sock";
+@unlink($path);
```

Comment:
Please move the `--CLEAN--` section between the `--FILE--` and `--EXPECTF--` sections.